### PR TITLE
Improve wording in instructions in Make Dead Links challenge

### LINKED ISSUE #11297: Change instructions in "Make Dead Links using the Hash Symbol"
#### Make Dead Links using the Hash Symbol

https://www.freecodecamp.com/challenges/make-dead-links-using-the-hash-symbol

#### Issue Description
Having spent some time in the help room, I frequently see many new campers trip on this challenge. Although the current instructions are not in any way vague or difficult to understand, there is something about it that leads most campers to come up with this incorrect code instead - 

`
<p>Click here for <a #="http://www.freecatphotoapp.com">cat photos</a>.</p>`

Along with not reading the instructions carefully, I think this also stems, in part, from the way this instruction is worded 

> 
Replace the value of your a element's href attribute with a #, also known as a hash symbol, to turn it into a dead link.

I feel perhaps the wording (`element`, `attribute`, `value`) comes across as too technical for some new campers, which leads them to think that the `href` attribute itself should be replaced with a `#`. Maybe it could be replaced with something like this 

> The current value of the href attribute is a link that points to "http://www.freecatphotoapp.com". Change the value of href attribute such that it is a hash (#) instead.

Also, in a previous challenge, there is a [diagram](https://i.imgur.com/hviuZwe.png) that depicts the various parts of a `p` element and an anchor tag. 

hviuzwe

While this may or may not be related to the issue here, the last arrow is somewhat incorrect - it actually points to the **value** of the **href attribute**, while the label reads **anchor tag's href attribute**.  Maybe a change in the label could be helpful for this issue? Thanks to @dhcodes for pointing this out!

Just putting in my two cents here! I think this is something we could look into. If the mods feel so too, I would be happy to work on a PR.

Cheers!

### PR DESCRIPTION
**Pull request by Manish-Giri**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #11297 
#### Description

Changed instructions in the challenge to clarify that the **value** of the `href` attribute needs to be replaced with a `#`, not the `href` itself.

closes #11297

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 1]
```diff
@@ -1741,7 +1741,7 @@
       "description": [
         "Sometimes you want to add <code>a</code> elements to your website before you know where they will link.",
         "This is also handy when you're changing the behavior of a link using <code>jQuery</code>, which we'll learn about later.",
-        "The <code>href</code> attribute is the quoted URL link within the <code>a</code> element. Replace the link value of your <code>a</code> element's <code>href</code> attribute with a <code>#</code>, also known as a hash symbol, to turn it into a dead link."
+        "The current value of the <code>href</code> attribute is a link that points to \"http://www.freecatphotoapp.com\". Replace this link with a <code>#</code>, also known as a hash symbol, to create a dead link."
```
Comment (Bouncey) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: ~~Replace this link with a <code>#</code>~~

Replace the <code>href</code> attribute value with a <code>#</code>

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: seed/challenges/01-front-end-development-certification/html5-and-css.json (hunk 1 above)
Comment (Bouncey) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: You have shown the camper where the `href` attr is by pointing out what the value is. You then call it a link, which not only is a change in terminology, it is also incorrect. 

The anchor element is the `link`, the `href` attr is the `target` of the `link`.

**Comment by Manish-Giri** [comment]
@Bouncey Thanks for the review! I made the changes you'd asked for.

**Review by Bouncey — APPROVED** [review]

**Comment by Bouncey** [comment]
## LGMT :+1: 

Thanks for contributing :tada:

**Pull request merged**